examples/minimal/packages/contracts/src/systems/ChatSystem.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+import { console } from "forge-std/console.sol";
+import { System } from "@latticexyz/world/src/System.sol";
+import { MessageTable } from "../codegen/Tables.sol";
+
+bytes32 constant SingletonKey = bytes32(uint256(0x060D));
+
+contract ChatSystem is System {
+  function sendMessage(string memory message) public returns (uint32) {
+    MessageTable.setEphemeral(SingletonKey, message);
+  }
+}
```

Comment:
The `SingletonKey` pattern was confusing to folks at the last hackathon (esp. since tables also support `singleton: true` for "real singleton tables"). Should we just use the `msg.sender` as key? Then the frontend could even display the sender of the message.

Reply:
I think we should stop exposing the SingletonKey to user and surface APIs contract and client side to interact with Singleton (we already do have the API tablegen side, we need it RECS / mud-react side too)